requirements/dev.txt

```diff
@@ -10,4 +10,5 @@ recommonmark>=0.5.0
 twine>=3.1.1  # for upload to pypi
 autodocsumm>=0.1.11
 pandas>=0.23.4
+pyomo>=5.7
```

Comment:
please remove the requirements from dev and main, and add it instead to bench.txt, it should be enough